web/annotation_layer_builder.css

```
@@ -15,6 +15,10 @@
 
 :root {
   --annotation-unfocused-field-background: url("data:image/svg+xml;charset=UTF-8,<svg width='1px' height='1px' xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' style='fill:rgba(0, 54, 255, 0.13);'/></svg>");
+  --inner-focus-line-color: Highlight;
+  --input-unfocus-border-color: transparent;
```

For consistency, since we use "unfocused" in a pre-existing variable in this file:
```suggestion
  --input-unfocused-border-color: transparent;
```